packages/now-node/test/fixtures/04-tsx-resolve/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "strict": true,
     "esModuleInterop": true,
     "sourceMap": true,
+    "jsx": "react",
```

Comment:
This is not necessary for this PR but TS was printing a warning and it confused me so I fixed it 😄 


> index.ts(1,17): error TS6142: Module './dep' was resolved to '/vercel/2df33c57/dep.tsx', but '--jsx' is not set.